backend/src/main/kotlin/app/ehrenamtskarte/backend/common/webservice/WebService.kt

```diff
@@ -77,10 +77,7 @@ class WebService {
 
         app.get("/health") { ctx -> healthHandler.handle(ctx) }
 
-        // TODO remove the condition once the Koblenz project is ready to go live
-        if (!production) {
-            app.post("/users/import") { ctx -> userImportHandler.handle(ctx) }
-        }
+        app.post("/users/import") { ctx -> userImportHandler.handle(ctx) }
```

Comment:
:thought_balloon: Personally I dislike abbreviations like this, as it makes it harder to read if you are not familiar with specific terms or the project in general. Especially in kotlin, since there is the default `it` available which makes the code more concise.
```suggestion
        app.post("/users/import") { userImportHandler.handle(it) }
```
If explicit naming adds anything or is necessary due to shadowing, the full word should be used imo:
```suggestion
        app.post("/users/import") { context -> userImportHandler.handle(context) }
```
Since this is nothing you introduced in this PR, feel free to ignore, just wanted to dump this somewhere :D

Reply:
I'm too lazy to change it here, but I'll update it in my other pull-request, along with other comments :)

Reply:
I'm too lazy to change it here, but I'll push that in my other pull-request, together with some other requested changes